web/src/scheduled_messages.js

```diff
@@ -181,8 +166,69 @@ export function get_count() {
 }
 
 export function get_filtered_send_opts(date) {
+    const st = compute_later_send_times();
+
     const day = date.getDay(); // Starts with 0 for Sunday.
     const hours = date.getHours();
+
+    const send_later_today = {
+        today_nine_am: {
+            text: $t(
+                {defaultMessage: "Today at {time}"},
+                {time: timerender.get_localized_date_or_time_for_format(st.today_nine_am, "time")},
+            ),
+            stamp: st.today_nine_am,
```

Comment:
`send_times` would be a better variable name for this.

Reply:
(The reasoning being that `compute_later_send_times` or even `compute_send_times` should return `send_times` -- the more times you rename a value in the codebase, the harder it is to read. And we have a general "no abbreviations" policy naming policy -- they tend to reduce readability, and editor autocomplete can eliminate the burden of typing extra characters).

Reply:
Wholly agreed. It's my custom and practice to use plurals on collections of things, but I slipped on that here.